tests/callbacks/test_callbacks.py

```diff
@@ -214,8 +214,6 @@ def on_test_end(self, trainer, pl_module):
 
 def test_early_stopping_no_val_step(tmpdir):
     """Test that early stopping callback falls back to training metrics when no validation defined."""
-    tutils.reset_seed()
-
     class ModelWithoutValStep(LightTrainDataloader, TestModelBase):
```

Comment:
@Borda shouldn't this be replaced by the new test model and set val_step=None?

Reply:
in this one I have just replaced the `tutils.default_model`, the others are coming in followup PRs as I wanted to keep it per blocks not all at once